src/components/gauge/gauge.tsx

```diff
@@ -111,13 +109,16 @@ export class Gauge implements GxComponent {
     return ((this.value - this.minValue) * 100) / this.calcTotalValues();
   }
 
-  private renderCircle(childRanges) {
+  private renderCircle(
+    childRanges: Array<HTMLGxGaugeRangeElement>
+  ): HTMLElement {
     const FULL_CIRCLE_RADIO = 100 / 2;
     const svgRanges = [];
+    const labelsRanges = [];
     const ONE_PERCENT_OF_CIRCLE_DREGREE = 3.6;
     const radius = FULL_CIRCLE_RADIO - this.thickness / 2;
 
-    function renderSvgCircle(currentChild, position, component): HTMLElement {
+    function renderSvgCircle(currentChild, position, component): SVGElement {
```

Comment:
This function doesn't seem to return SVGElement, but JSX. It's ok if you leave it without specifying a return type.
BTW, why not declare this function as a method and instead of passing the component as the third parameter, use `this`?

Reply:
I was planning to use the `bind()` method to change the reference of  `this`. But yes! I defined all those "sub" functions and declared it as a method of the component.
Change all this in 65fb97b